### PR TITLE
[ashell] Removing extra window info and various cleanup

### PR DESCRIPTION
**Pull request by brianjjones**
Signed-off-by: Brian J Jones <brian.j.jones@intel.com>

### CODE REVIEW

**Comment by poussa** [comment]
+1

File: src/ashell/comms-shell.c [hunk 1]
```diff
@@ -697,4 +697,4 @@ void shell_unit_test()
     }
     printf("All tests were successful \n");
 }
-#endif
+#endif //CONFIG_SHELL_UNIT_TESTS
```
Comment (grgustaf) [inline src/ashell/comms-shell.c]: To follow convention, two spaces before //, one space after.

**Comment by grgustaf** [comment]
Two more things I noticed running it:
- We should probably still print a short message indicating that the real shell is at ACM0 or so
- We are currently running the HelloWorld.js (or whatever is passed in with JS=) even though we're in ashell mode, that should be disabled I think.

**Comment by brianjjones** [comment]
@grgustaf True no real reason (I think) to be running hello world, its just the default behaviour when JS isn't passed.  I'll look into adding the removal of helloworld in a future patch

**Comment by brianjjones** [comment]
@grgustaf Warning added

**Comment by grgustaf** [comment]
+1

**Pull request merged**